apps/web/src/views/NFTs/NFTs.tsx

```diff
@@ -1,9 +1,52 @@
-import { Flex } from "@chakra-ui/react";
+import { Flex, Heading, Icon, SimpleGrid } from "@chakra-ui/react";
+import { fullId, sortedByLastUpdate } from "@umami/core";
+import { useCurrentAccount, useGetAccountNFTs } from "@umami/state";
+import BigNumber from "bignumber.js";
 
+import { NFTCard } from "./NFTCard";
+import { FilterIcon } from "../../assets/icons";
 import { EmptyMessage } from "../../components/EmptyMessage";
+import { useColor } from "../../styles/useColor";
 
-export const NFTs = () => (
-  <Flex height="full">
-    <EmptyMessage subtitle="NFTs" title="NFT" />
-  </Flex>
-);
+export const NFTs = () => {
+  const color = useColor();
+  const account = useCurrentAccount()!;
+  const nfts = sortedByLastUpdate(useGetAccountNFTs()(account.address.pkh));
+  const totalCount = nfts.reduce((acc, nft) => acc.plus(nft.balance), BigNumber(0)).toNumber();
+
+  let gridTemplateColumns = "repeat(auto-fit, minmax(min(100%/2, max(157px, 100%/4)), 1fr))";
+  if (nfts.length < 3) {
+    gridTemplateColumns = `repeat(auto-fit, min(100% / ${nfts.length} - 6px, 100%))`;
```

Comment:
What do we calculating here `${nfts.length} - 6px`?

Reply:
without it the 2nd image doesn't fit and wraps around to the 2nd row
<img width="806" alt="Screenshot 2024-08-05 at 16 25 42" src="https://github.com/user-attachments/assets/01a28e45-ffae-433a-8b8b-045b1b64624f">